02_configure_host.sh

```diff
@@ -129,7 +129,6 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
     if [ "$INT_IF" ]; then
         echo -e "DEVICE=$INT_IF\nTYPE=Ethernet\nONBOOT=yes\nNM_CONTROLLED=no\nBRIDGE=${BAREMETAL_NETWORK_NAME}" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-$INT_IF
         if [[ $EXTERNAL_SUBNET =~ .*:.* ]]; then
-             sudo firewall-cmd --zone=libvirt --add-service=dhcpv6-client
```

Comment:
Should just be a metal3-dev-env default, added here:
  https://github.com/metal3-io/metal3-dev-env/pull/251